test/e2e/examples2_test.go

```diff
@@ -86,16 +86,25 @@ func (suite *ExamplesTestSuite2) TestServiceTypesExample() {
 }
 
 func (suite *ExamplesTestSuite2) TestSimpleProdWithVolumes() {
+	if skipESExternal {
```

Comment:
Why is this being skipped? Is it be because the example needs an ES to exist at a specific example?

Reply:
Yes, the examples are hard coded to use an external instance.

Reply:
Could you add that as a comment to the code?

Reply:
I see you did that already :-)